packages/snaps-controllers/src/snaps/location/npm.test.ts

```diff
@@ -66,11 +66,11 @@ describe('NpmLocation', () => {
     const manifest = await location.manifest();
     const sourceCode = (
       await location.fetch(manifest.result.source.location.npm.filePath)
-    ).value.toString();
+    ).toString();
     assert(manifest.result.source.location.npm.iconPath);
     const svgIcon = (
       await location.fetch(manifest.result.source.location.npm.iconPath)
-    ).value.toString();
+    ).toString();
```

Comment:
Why are these changes necessary?

Reply:
Because we are forcing everything to be `Uint8Array` now and `Uint8Array.toString()` does not encode by default.

My guess is that this probably was a Buffer or something else under test and so it was encoded correctly before, but isn't now that the `value` is guaranteed to be a `Uint8Array`.

Our virtual file implementation has a nicer implementation that handles encoding, so seemed simple to just use that.